### PR TITLE
Fix queue handling in K8sIntegration and k8s_glue_example.py

### PR DESCRIPTION
**Pull request by FeU-aKlos**
Facilitate, that in the clearml-agent helm chart, multiple queues can be assigned with following format: 'queue1,queue2'.

### CODE REVIEW

File: examples/k8s_glue_example.py [hunk 1]
```diff
@@ -91,7 +95,10 @@ def k8s_user_props_cb(pod_number=0):
             ssh_port_number=args.ssh_server_port) if args.ssh_server_port else None,
         namespace=args.namespace, max_pods_limit=args.max_pods or None,
     )
-    k8s.k8s_daemon(args.queue, use_owner_token=args.use_owner_token)
+    if "," in args.queue:
```
Comment (jkhenning) [inline examples/k8s_glue_example.py:98]: This seems too elaborate, you can simply do:
```python
args.queue = [q.strip() for q in args.queue.split(",") if q.strip()]
```
Which would also handle input such as `"queue1,"`

File: docker/k8s-glue/glue-build-gcp/Dockerfile [hunk 2]
```diff
@@ -8,15 +8,18 @@ ENV LANG=en_US.UTF-8
...
 
-COPY ../build-resources/entrypoint.sh /root/entrypoint.sh
+COPY ./entrypoint.sh /root/entrypoint.sh
 COPY ./provider_entrypoint.sh /root/provider_entrypoint.sh
-COPY ./build-resources/k8s_glue_example.py /root/k8s_glue_example.py
+COPY ./k8s_glue_example.py /root/k8s_glue_example.py
 COPY ./clearml.conf /root/clearml.conf
 
+#TODO: should be adjusted
+RUN python3 -m pip uninstall clearml-agent -y && python3 -m pip install git+https://github.com/FeU-aKlos/clearml-agent.git@feature/support_multi_queeus_with_k8s
```
Comment (FeU-aKlos) [inline docker/k8s-glue/glue-build-gcp/Dockerfile:23]: there is still the dockerimage linked, which i created my self, I was not sure how to handle the thing, that the clearml-agent python package should be installed, for usage in the helm chart, without the actual (official) repo having the changes. Any Ideas on that? Should i adjust the setup.sh script and reference my fork there for pip install or the main branch, if i expect, that this PR will be accepted...

Comment (FeU-aKlos) [reply docker/k8s-glue/glue-build-gcp/Dockerfile:23]: or in the entrypoint.sh

Comment (jkhenning) [reply docker/k8s-glue/glue-build-gcp/Dockerfile:23]: Don't leave your repo here - once we merge the PR we'll build a new image and publish it

Comment (FeU-aKlos) [reply docker/k8s-glue/glue-build-gcp/Dockerfile]: Okay, great, thanks for clarification

**Pull request merged**